components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/Blocks/transaction-life-cycle.adoc

```diff
@@ -14,6 +14,7 @@ A transaction is submitted to one of the gateways, functioning as the Mempool an
 
 === 2. Validation
 The Mempool performs validation on the transaction to check for its validity. If the transaction is invalid, it will not proceed to the next stages.
```

Comment:
```suggestion
The Mempool performs validation on the transaction to check for its validity. If the transaction is invalid, it will not proceed to the next stages.

```